edge/pkg/metamanager/process.go

```diff
@@ -175,7 +176,7 @@ func (m *metaManager) processUpdate(message model.Message) {
 		resp := message.NewRespByMessage(&message, OK)
 		sendToEdged(resp, message.IsSync())
 	case cloudmodules.EdgeControllerModuleName, cloudmodules.DynamicControllerModuleName:
-		sendToEdged(&message, message.IsSync())
```

Comment:
Should not modify the `message.IsSync()`?

Reply:
yes, typo, corrected